admin/autoupdater/files/usr/sbin/autoupdater

```diff
@@ -41,6 +43,9 @@ local function parse_args()
   while arg[i] do
     if arg[i] == '-f' then
       force = true
+    elseif arg[i] == '-F' then
```

Comment:
ah, read it too late .. wont make any big difference, while autoupdater will be replaced - as you said